js/oceanex.js

```diff
@@ -0,0 +1,323 @@
+'use strict';
+
+//  ---------------------------------------------------------------------------
+
+const Exchange = require ('./base/Exchange');
+const { ExchangeError } = require ('./base/errors');
+
+module.exports = class oceanex extends Exchange {
+    describe () {
+        return this.deepExtend (super.describe (), {
+            'id': 'oceanex',
+            'name': 'oceanex',
+            'countries': [ 'SG' ],
+            'urls': {
+                'api': 'https://api.oceanex.pro/v1/',
+                'www': 'https://www.oceanex.pro.com',
+                'doc': 'https://api.oceanex.pro/doc/v1/',
+            },
+            'api': {
+                'public': {
+                    'get': [
+                        'markets',
+                        'tickers/{pair}/',
+                        'tickers_multi',
```

Comment:
If the exchange has `tickers_multi`, we will need the implementation for `fetchTickers` as well: 

- https://github.com/ccxt/ccxt/blob/master/js/hitbtc2.js#L796
- https://github.com/ccxt/ccxt/blob/master/js/kucoin.js#L399
- https://github.com/ccxt/ccxt/blob/master/js/huobipro.js#L372